gammapy/modeling/models/cube.py

```diff
@@ -373,7 +379,9 @@ def integrate_geom(self, geom, gti=None, oversampling_factor=None):
             )
 
         if self.temporal_model:
-            integral = self.temporal_model.integral(gti.time_start, gti.time_stop)
+            integral = self.temporal_model.integral(
```

Comment:
here I would rather compute the integral over time and then compute the integral of the spectral model.